src/libs/kata-types/src/annotations/mod.rs

```diff
@@ -316,6 +316,13 @@ pub const KATA_ANNO_CFG_HYPERVISOR_PREFETCH_FILES_LIST: &str =
 pub const KATA_ANNO_CFG_SANDBOX_BIND_MOUNTS: &str =
     "io.katacontainers.config.runtime.sandbox_bind_mounts";
 
+/// A sandbox annotation to specify the type of machine being emulated by the hypervisor.
+pub const KATA_ANNO_HYPERVISOR_MACHINE_TYPE: &str = "io.katacontainers.hypervisor.machine_type";
+/// A sandbox annotation for passing the default vcpus assigned for a VM by the hypervisor.
+pub const KATA_ANNO_HYPERVISOR_DEFAULT_VCPUS: &str = "io.katacontainers.hypervisor.default_vcpus";
+/// A sandbox annotation for the memory assigned for a VM by the hypervisor.
+pub const KATA_ANNO_HYPERVISOR_DEFAULT_MEMORY: &str = "io.katacontainers.hypervisor.default_memory";
```

Comment:
How are those annotation different from the existing `io.katacontainers.config.hypervisor.{machine_type,default_vcpus,default_memory}`?  Shouldn't we be using those?